Source/KSCrash/Recording/Tools/KSDynamicLinker.c

```diff
@@ -381,6 +381,16 @@ static void getCrashInfo(const struct mach_header* header, KSBinaryImage* buffer
         KSLOG_DEBUG("Found second message: %s", crashInfo->message2);
         buffer->crashInfoMessage2 = crashInfo->message2;
     }
+    if(isValidCrashInfoMessage(crashInfo->message2))
```

Comment:
You should check `backtrace` here

Reply:
oh, you are right. must be a copy paste bug